azext_iot/tests/helpers.py

```diff
@@ -8,10 +8,46 @@
 import os
 
 from inspect import getsourcefile
+from azure.iot.device import ProvisioningDeviceClient, IoTHubDeviceClient
 
 from azext_iot.common.utility import read_file_content
 
+GLOBAL_PROVISIONING_HOST = "global.azure-devices-provisioning.net"
+
 
 def load_json(filename):
     os.chdir(os.path.dirname(os.path.abspath(getsourcefile(lambda: 0))))
     return json.loads(read_file_content(filename))
+
+
+def dps_connect_device(device_id: str, credentials: dict) -> IoTHubDeviceClient:
```

Comment:
Powerful helper overall in combination with device client. I think we'll definitely be interested in leveraging the device SDK more with device simulation capabilities.